webviz_subsurface/plugins/_pvt_plot/_views/_pvt/_utils/_plot_utils.py

```diff
@@ -89,10 +89,10 @@ def create_traces(
         data_frame = data_frame.loc[data_frame["KEYWORD"] == "PVTW"]
         dim_column_name = "PRESSURE"
 
-    data_frame = data_frame.sort_values(
-        ["PRESSURE", "VOLUMEFACTOR", "VISCOSITY"],
-        ascending=[True, True, True],
-    )
+    # data_frame = data_frame.sort_values(
+    #     ["PRESSURE", "VOLUMEFACTOR", "VISCOSITY"],
+    #     ascending=[True, True, True],
+    # )
```

Comment:
If I remember correctly this was only to ensure that the list is sorted and that the points plotted are connected with lines in the correct order (under the assumption that Plotly is plotting in the order given by the input array). This might not have an effect since the data frame might already be sorted beforehand. However, I wonder if the plots would still be correct if the df wasn't. If we are 100% sure that the df is always sorted correctly, this can be removed.